test/unit/remote-config/remote-config.spec.ts

```diff
@@ -612,7 +642,11 @@ describe('RemoteConfig', () => {
           expect(cond.tagColor).to.equal(TagColor.BLUE);
 
           const parsed = JSON.parse(JSON.stringify(template));
-          expect(parsed).deep.equals(REMOTE_CONFIG_RESPONSE);
+          const templateCopy = deepCopy(REMOTE_CONFIG_RESPONSE);
```

Comment:
I had a little trouble following the logic here. May be rename the variables to `expectedTemplate` and `expectedVersion`.

Reply:
Ah good call! Thanks!